envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/SnapshotDebugTest.kt

```diff
@@ -97,4 +97,43 @@ open class SnapshotDebugTest : EnvoyControlTestConfiguration() {
         // then
         assertThat(snapshot.found).isFalse()
     }
+
+    @Test
+    open fun `should return global snapshot debug info from xds`() {
+        // given
+        registerService(name = "echo")
```

Comment:
Is this needed for the test? If we register service "echo", we should use it and check if "echo" cluster and/or endpoint is in the snapshot. Or we should get rid of this line.